doc/release-notes.md

```diff
@@ -36,377 +31,124 @@ frequently tested on them.
 
 ### :exclamation::exclamation::exclamation: MacOS 10.13 High Sierra :exclamation::exclamation::exclamation:
 
-**Currently there are issues with the 3.0.0 gitian release on MacOS version 10.13 (High Sierra), no reports of issues on older versions of MacOS.**
+**Currently there are issues with the 3.0.0+ gitian release on MacOS version 10.13 (High Sierra), no reports of issues on older versions of MacOS.**
 
- 
 Notable Changes
 ==============
 
 zPIV Updates
 --------------
 
-### zPIV Staking
-
-zPIV Staking is now live! With the release of zPIV staking, there are effectively 2 versions of zPIV, zPIV minted on the 3.0.6 PIVX wallet or lower, and zPIV minted on PIVX wallet version or higher. New features in this release will require the use of zPIV v2, zPIV minted on this wallet release or later. If you currently hold zPIV v1 and wish to take advantage of zPIV staking and deterministic zPIV, you will need to spend the zPIV v1 to yourself and remint zPIV v2.
-In order to take advantage of zPIV staking, you must mint new zPIV v2 and wait for at least 200 confirmations before that zPIV is considered valid for staking.
-
-**Note:**  To find your zPIV version, click the privacy tab, then the zPIV Control button then expand the arrows next to the desired denomination. 
-
-
-### Deterministic zPIV Seed Keys
-
-zPIV is now associated with a deterministic seed key. With this seed key, users are able to securely backup their zPIV outside of the wallet that the zPIV had been minted on. zPIV can also be transferred from wallet to wallet without the need of transferring the wallet data file.
-
-
-### Updated zPIV minting
-
-zPIV minting now only requires 1 further mint (down from 2) to mature. zPIV mints still require 20 confirmations.  Mints also require that the 'second' mint is at least two checkpoints deep in the chain (this was already the case, but the logic was not as precise).
+### Fix spending for v1 zPIV created before block 1050020
 
+The transition to v2 zPIV and reset of the accumulators caused blocks 1050000 - 1050010 to be accumulated twice. This was causing a number v1 zPIV to not create valid witnesses, and thus were not spendable. This problem is fixed by double accumulating blocks 1050000-1050010 when creating the witness. Any user that had issues spending zPIV v1 will now be able to convert that into PIV and then zPIV v2 (if desired).
 
-### zPIV Search
+### Adjustment to staking properties to reduce orphaned blocks
 
-Users will now have the ability to search the blockchain for a specific serial # to see if a zPIV denomination has been spent or not.
+zPIV stake set to update more frequently and lowering the stake hashdrift to 30 seconds to improve number of orphans being experienced by PIVX stakers.
```

Comment:
I'd rather "reduce" the number of orphans than improve them ;-)

Reply:
fixed